<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findwindow</class>
 <widget class="QMainWindow" name="findwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>员工后台管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="photo.qrc">
    <normaloff>:/2.png</normaloff>:/2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#findwindow
{
border-image: url(:/5.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>查询全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findOneButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>精准查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>主菜单</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findIdEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入查询ID</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="RollText" name="widgetAll" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="textlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollText</class>
   <extends>QWidget</extends>
   <header location="global">rolltext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
